ext/rubymain.cpp

```diff
@@ -88,7 +91,7 @@ static inline VALUE ensure_conn(const uintptr_t signature)
 t_event_callback
 ****************/
 
-static inline void event_callback (struct em_event* e)
+static inline VALUE event_callback (struct em_event* e)
```

Comment:
I changed the contract here to return the symbol of the method that was called so I could put that in the tick timing.